src/index.html

```diff
@@ -1,15 +1,162 @@
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="data">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off">
+        </div>
+
+        <div class="data">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off">
+        </div>
+
+        <div class="data">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            class="ageline"
+            name="age"
+            id="age"
+            value="12"
+            min="12"
+            max="110">
+        </div>
+
+        <div class="data">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            name="date"
+            id="date">
+        </div>
+
+        <div class="data">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="term"
+            id="terms">
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="data">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com."
+            required autocomplete="off">
+        </div>
+
+        <div class="data">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="15"
+            autocomplete="off">
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="data"> Do you love cats?
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            id="yes"
+            name="choice"
+            value="yes">
+          <label for="no">No</label>
+          <input
+            type="radio"
+            name="choice"
+            id="no"
+            value="no">
+        </div>
```

Comment:
give radio buttons group a more specific name like 'love-cats' (yes or no)
```suggestion
        <div class="data"> Do you love cats?
          <label for="yes">Yes</label>
          <input
            type="radio"
            id="yes"
            name="love-cats"
            value="yes">
          <label for="no">No</label>
          <input
            type="radio"
            name="love-cats"
            id="no"
            value="no">
        </div>
```